docs/dataservice-write-example.md

```diff
@@ -34,6 +34,39 @@ To execute the example, run:
 
 If everything is fine, you would see `Publish Successful - TraceID: <TraceId generated by the platform>`.
 
+### Building on Android
+
+Prerequsities: make sure that you have correctly setup the Android environment (downloaded both Android SDK and NDK from official site). Also, make sure that your provided correct keyId, keySecret, and other information stated in the beginning of this README file.
```

Comment:
Make Prerequsities a h3
the other two points as unordered list entries.

### Prerequsities
* Setup the Android environment ...
* Provider correct keyId ...

Reply:
Done